fastestimator/schedule/lr_shedule.py

```diff
@@ -24,19 +29,33 @@ def cosine_decay(time: int, cycle_length: int, init_lr: float, min_lr: float = 1
     fe.Estimator(..., traces=[s])
     ```
 
+    For more information, checkout SGDR: https://arxiv.org/pdf/1608.03983.pdf.
+
     Args:
         time: The current step or epoch during training starting from 1.
         cycle_length: The decay cycle length.
         init_lr: Initial learning rate to decay from.
         min_lr: Minimum learning rate.
         start: The step or epoch to start the decay schedule.
+        cycle_multiplier: The factor of which next cycle length will be multiplied.
```

Comment:
The factor by which the next cycle length will be multiplied.

Reply:
y